pages/initiatives/gsoc/gsoc2023ideas.md

```diff
@@ -112,7 +113,7 @@ All ZAP projects will be mentored by the ZAP Project Leaders:
 ##### Getting Started
 
 * [BLT Core](https://github.com/owasp/blt/)
-* [Flutter App](https://github.com/Bugheist/Flutter/)
+* [BugHeist Mobile App](https://github.com/Bugheist/Flutter/)
```

Comment:
```suggestion
```

Reply:
I think we should keep a reference to the app under the previous "Flutter App" name.

Reply:
I think we have to remove any reference of Bugheist even if it's in the URL, so lets take it out to be on the safe side.

Reply:
How will people know where to look for contributing on the flutter app if they want?